src/libaktualizr/utilities/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(SOURCES dequeue_buffer.cc
 set(HEADERS config_utils.h
             dequeue_buffer.h
             exceptions.h
+            results.h
```

Comment:
Minor: it's a bit odd that `results.h` sits in `utilities`. It was already the case before with `events.h`, so it's not a new problem.

Reply:
Yes, I agree, but I couldn't find a better solution at the moment.